src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,76 @@
-export const Pagination = () => {};
+/* eslint-disable array-callback-return */
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  currentPage: number;
+  perPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  currentPage,
+  perPage,
+  onPageChange,
+}) => {
+  const pages = getNumbers(1, Math.ceil(total / perPage)).map(page => page);
+  const handlePageChange = (page: number) => {
```

Comment:
```suggestion
  const handlePageChange = (page: number) => () => {
```
better to return arrow function and just call in jsx